specification/storage/data-plane/Microsoft.FileStorage/stable/2024-11-04/file.json

```diff
@@ -6650,11 +6665,11 @@
         "format": {
           "type": "string",
           "enum": [
-            "SDDL",
-            "binary"
+            "Sddl",
+            "Binary"
```

Comment:
It seems that from sever's response, these fields are full lowcase like `sddl` and `binary`.